client/rest-high-level/src/main/java/org/elasticsearch/client/ml/NodeAttributes.java

```diff
@@ -147,4 +148,15 @@ public boolean equals(Object other) {
             Objects.equals(transportAddress, that.transportAddress) &&
             Objects.equals(attributes, that.attributes);
     }
+
+    @Override
+    public String toString() {
```

Comment:
Since this class `implements ToXContentObject` you should just call `Strings.toString(this)`. `Strings` supplies some nice helpers and is located in `org.elasticsearch.common`.

Reply:
Done.